api/src/main/java/org/bf2/operator/resources/v1alpha1/ServiceAccount.java

```diff
@@ -15,6 +16,7 @@
         editableEnabled = false
 )
 @ToString
+@EqualsAndHashCode
```

Comment:
This missing `equals` implementation is resulting in superfluous patch calls for each MK in the sync loop.